### ISSUE TITLE
deleting all trackcache is a memory hog

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#1823]
When deleting a large amount of cached data I get this error:

```
Caused by: org.codehaus.groovy.grails.web.servlet.mvc.exceptions.ControllerExecutionException: Runtime error executing action
	... 12 more
Caused by: java.lang.reflect.InvocationTargetException
	... 12 more
Caused by: org.springframework.orm.hibernate4.HibernateSystemException: Could not instantiate entity:  : org.bbop.apollo.TrackCache; nested exception is org.hibernate.InstantiationException: Could not instantiate entity:  : org.bbop.apollo.TrackCache
	at org.grails.datastore.gorm.GormStaticApi$_methodMissing_closure2.doCall(GormStaticApi.groovy:102)
	at org.bbop.apollo.TrackController$__tt__clearOrganismCache_closure11.doCall(TrackController.groovy:73)
	at org.bbop.apollo.TrackController.$tt__clearOrganismCache(TrackController.groovy:71)
	... 12 more
```

**Comment by nathandunn** [comment]
Replace:

- [ ] https://github.com/GMOD/Apollo/blob/b47843478759f6e3e746744911335bc972f8dd01/grails-app/controllers/org/bbop/apollo/TrackController.groovy#L58
- [ ] https://github.com/GMOD/Apollo/blob/b47843478759f6e3e746744911335bc972f8dd01/grails-app/controllers/org/bbop/apollo/TrackController.groovy#L73
- [ ] https://github.com/GMOD/Apollo/blob/b47843478759f6e3e746744911335bc972f8dd01/grails-app/controllers/org/bbop/apollo/TrackController.groovy#L83

**Comment by nathandunn** [comment]
Fixed as part of #1824
